app/services/auth_hash.rb

```diff
@@ -26,10 +27,16 @@ def token
     user_hash.fetch('credentials', {}).fetch('token')
   end
 
+  def site_admin
+    return true if non_staff_github_admins_ids.include?(uid)
+    raw_info[:site_admin]
+  end
+
   private
 
-  def info
-    user_hash.fetch('info', {})
+  def non_staff_github_admins_ids
+    return [] unless ENV['NON_STAFF_GITHUB_ADMIN_IDS'].present?
+    ENV['NON_STAFF_GITHUB_ADMIN_IDS'].split(',')
```

Comment:
You probably wants a `compact` here to make sure `nil` never ends up in that array. I'd also add a test case to make sure setting the environment variable to `,,1` does not include a nil in the resulting array.